packages/client/hmi-client/src/services/models/simulation-service.ts

```diff
@@ -47,7 +47,17 @@ export async function getSimulation(id: Simulation['id']): Promise<Simulation |
 
 export async function createSimulation(simulation: Simulation): Promise<Simulation | null> {
 	try {
-		const response = await API.post('/simulation', simulation);
+		// const response = await API.post('/simulation', simulation);
+		const request = {
+			simulator_id: 1,
```

Comment:
Why do you need to send a `simulator_id`?

Reply:
It's simply a required field by data-service